tsconfig.json

```diff
@@ -18,6 +18,9 @@
     // Exclude tests as they have their own tsconfigs.
     "e2e",
     "e2e-testing",
+    // Exclude `testUtils` which has `vitest` imports to avoid conflicting types
+    // for the `chai` npm package.
+    "src/testUtils.ts",
```

Comment:
Might be clearer to have it in a folder in case you want to break it down into separate files.